bigtable/quickstart/main.py

```diff
@@ -19,7 +19,8 @@
 from google.cloud import bigtable
 
 
-def main(project_id, instance_id, table_id):
+def main(project_id="project-id", instance_id="instance-id",
+    table_id="my-table"):
```

Comment:
This probably fails flake8. Needs `table_id` needs to be aligned with `project_id` in the line above.

```
def main(project_id="project-id", instance_id="instance-id",
         table_id="my-table"):
```

Reply:
Fixed before you commented this 😛 IntelliJ autoformat typically is better about this stuff 🤷‍♂️